sys/test/test.txt

```diff
@@ -733,6 +733,7 @@ mutate5(filename ptr[in, filename], flags flags[open_flags]) fd
 mutate6(fd fd, data ptr[in, array[int8]], size bytesize[data])
 mutate7(a0 ptr[in, string], a1 len[a0])
 mutate8(a0 proc[100, 4, opt])
+mutate9(a0 int32)
```

Comment:
Use one of the existing calls that accepts 1 int, there should be some. It will remove significant part of the diff in this change.